performance/tests/test_eqprs.py

```diff
@@ -0,0 +1,45 @@
+from django.test import TestCase
+from performance.admin import EQPRSFileUploadAdmin
+from django.contrib.auth.models import User
+from django.contrib.admin.sites import AdminSite
+from django.contrib import admin
+from performance.models import EQPRSFileUpload
+from django.test import RequestFactory
+
+USERNAME = "testuser"
+EMAIL = "testuser@domain.com"
+PASSWORD = "12345"
+
+
+def get_mocked_request(superuser):
+    request = RequestFactory().get('/get/request')
+    request.method = 'GET'
+    request.user = superuser
+    return request
+
+
+class EQPRSFileUploadTestCase(TestCase):
+    def setUp(self):
+        self.superuser = User.objects.create_user(
+            username=USERNAME,
+            password=PASSWORD,
+            is_staff=True,
+            is_superuser=True,
+            is_active=True,
+        )
+        self.mocked_request = get_mocked_request(self.superuser)
+
+    def test_selected_user_is_current_user(self):
+        ea = EQPRSFileUploadAdmin(EQPRSFileUpload, AdminSite())
+        form = ea.get_form(self.mocked_request)
+        default_user = form.base_fields['user'].queryset.first()
+
+        self.assertEqual(default_user, self.superuser)
```

Comment:
this is something we want to validate as set on the object after save, not something set client-side.

Reply:
that is because we don't want users to be able to mess with the value that's set here

Reply:
yea, I wanted to make sure the dropdown displayed the correct user as selected. now that we removed that dropdown, we don't need this test anymore